timeliney.typ

```diff
@@ -438,12 +439,18 @@
       }
     }
 
-    extratask = ((type: "task", name: title, lines: ((from: min, to: max),)),)
+    extratask = ((type: "task", name: title, lines: ((from: min, to: max, style: style),)),)
   }
 
   ((type: "taskgroup", tasks: extratask + tasks),)
 }
 
-#let milestone(body, at: none, ..options) = {
-  ((type: "milestone", at: at, body: body, ..options.named()),)
+#let milestone(body, at: none, emoji: none, ..options) = {
+  let display_body = if emoji != none {
+    emoji + " " + body
+  } else {
+    body
+  }
```

Comment:
I'm not particularly sure about the emoji, because it seems to be like it could simply be added manually to the body and wouldn't need a special option for it, though I'm open to being convinced otherwise.

Reply:
Sorry, I'm new to typst and still trying to understand how the script really works. 

You're right I was able to use the emoji within the text block itself, therefore I will rollback these changes.

<img width="1189" alt="image" src="https://github.com/user-attachments/assets/374df693-3ba0-4691-b5bc-405bdab023db" />

Reply:
Great!